catalog/spatial/wfs/spatial-wfs-featuretransformer-handlebars/src/test/java/org/codice/ddf/spatial/ogc/wfs/transformer/handlebars/HandlebarsWfsFeatureTransformerTest.java

```diff
@@ -287,6 +287,108 @@ public void xmlAttributesCanBeMapped() {
     assertThat(getAttributeValue(Core.METACARD_OWNER, metacard), is("value3"));
   }
 
+  @Test
+  public void resourceSizeInBytes() {
+    transformer.setAttributeMappings(
+        Collections.singletonList(
+            createMapping(Core.RESOURCE_SIZE, "ResourceSize", "{{ResourceSize}}")));
+    transformer.setDataUnit(WfsConstants.B);
+
+    final Optional<Metacard> metacardOptional = transformer.apply(inputStream, mockWfsMetadata);
+    assertThat(
+        "The transformer should have returned a metacard but didn't.",
+        metacardOptional.isPresent(),
+        is(true));
+
+    final Metacard metacard = metacardOptional.get();
+    assertThat(getAttributeValue(Core.RESOURCE_SIZE, metacard), is("1.0"));
+  }
+
+  @Test
+  public void resourceSizeInKilobytes() {
+    transformer.setAttributeMappings(
+        Collections.singletonList(
+            createMapping(Core.RESOURCE_SIZE, "ResourceSize", "{{ResourceSize}}")));
+    transformer.setDataUnit(WfsConstants.KB);
+
+    final Optional<Metacard> metacardOptional = transformer.apply(inputStream, mockWfsMetadata);
+    assertThat(
+        "The transformer should have returned a metacard but didn't.",
+        metacardOptional.isPresent(),
+        is(true));
+
+    final Metacard metacard = metacardOptional.get();
+    assertThat(getAttributeValue(Core.RESOURCE_SIZE, metacard), is("1024.0"));
+  }
+
+  @Test
+  public void resourceSizeInMegabytes() {
+    transformer.setAttributeMappings(
+        Collections.singletonList(
+            createMapping(Core.RESOURCE_SIZE, "ResourceSize", "{{ResourceSize}}")));
+    transformer.setDataUnit(WfsConstants.MB);
+
+    final Optional<Metacard> metacardOptional = transformer.apply(inputStream, mockWfsMetadata);
+    assertThat(
+        "The transformer should have returned a metacard but didn't.",
+        metacardOptional.isPresent(),
+        is(true));
+
+    final Metacard metacard = metacardOptional.get();
+    assertThat(getAttributeValue(Core.RESOURCE_SIZE, metacard), is("1048576.0"));
+  }
+
+  @Test
+  public void resourceSizeInGigabytes() {
+    transformer.setAttributeMappings(
+        Collections.singletonList(
+            createMapping(Core.RESOURCE_SIZE, "ResourceSize", "{{ResourceSize}}")));
+    transformer.setDataUnit(WfsConstants.GB);
+
+    final Optional<Metacard> metacardOptional = transformer.apply(inputStream, mockWfsMetadata);
+    assertThat(
+        "The transformer should have returned a metacard but didn't.",
+        metacardOptional.isPresent(),
+        is(true));
+
+    final Metacard metacard = metacardOptional.get();
+    assertThat(getAttributeValue(Core.RESOURCE_SIZE, metacard), is("1073741824.0"));
```

Comment:
Where do these data sizes come from? Are they just magic numbers returned from the mock?

Reply:
They come from https://github.com/connexta/gsr-yorktown/pull/1881. I'll update the tests to use the constants so it's clear where the numbers are coming from.